assets/src/edit-story/elements/index.js

```diff
@@ -44,9 +44,19 @@ export const createNewElement = (type, attributes = {}) => {
 };
 
 export const createPage = (attributes = {}) => {
-  const { elements, backgroundElementId } = attributes;
+  const { elements: oldElements, ...rest } = attributes;
+
+  // Ensure all existing elements get new ids
+  const elements = (oldElements || []).map(({ type, ...attrs }) =>
+    createNewElement(type, attrs)
+  );
+  const newAttributes = {
+    elements,
+    ...rest,
+  };
+
   // Enforce having background element for each Page.
-  if (!backgroundElementId) {
+  if (!newAttributes.backgroundElementId) {
```

Comment:
Shouldn't there be an else here as well?

Reply:
Oh yeah, the existing background element will have a new id as well, and I just changed it so it doesn't match. Well done, me.

I'll add some tests to make sure all the cases are covered here.

Reply:
Fixed in 5ad3bc17 with test.